sdk/attestation/attestation/package.json

```diff
@@ -114,7 +114,7 @@
     "chai-as-promised": "^7.1.1",
     "cross-env": "^7.0.2",
     "dotenv": "^8.2.0",
-    "downlevel-dts": "~0.4.0",
+    "downlevel-dts": "^0.8.0",
```

Comment:
Packages with major version 0 can have breaking changes between minor versions Should we be using ~ instead of ^ due to this?

Reply:
Good catch, I updated semvers to use `~` instead.

Reply:
This is not necessary:

https://www.npmjs.com/package/semver#caret-ranges-123-025-004

> Allows changes that do not modify the left-most non-zero element in the [major, minor, patch] tuple. In other words, this allows patch and minor updates for versions 1.0.0 and above, patch updates for versions 0.X >=0.1.0, and no updates for versions 0.0.X.

> Many authors treat a 0.x version as if the x were the major "breaking-change" indicator.

> Caret ranges are ideal when an author may make breaking changes between 0.2.4 and 0.3.0 releases, which is a common practice. However, it presumes that there will not be breaking changes between 0.2.4 and 0.2.5. It allows for changes that are presumed to be additive (but non-breaking), according to commonly observed practices.

Reply:
TIL, thanks @xirzec!